docs/api/1.x/index.rst

```diff
@@ -17,7 +17,6 @@ Full detailed API documentation:
    records
    groups
    permissions
-   synchronisation
```

Comment:
Is this a mistake or do you think the sync docs are not relevant ? https://github.com/Kinto/kinto/blob/master/docs/api/1.x/synchronisation.rst